src/assets/styles/smp.scss

```diff
@@ -219,7 +222,8 @@
 }
 
 // BP 3
-@media (min-width: 600px) and (max-width: 959px) {
+@media (width >= 600px) and (width <= 959px) {
```

Comment:
unsure why these have been changed from `min-width` to `width`. Usually media queries are defined in terms of max and min dimensions no?